### DISCUSSION TITLE
registration without reslicing/changing dimensions of moved/registered image

### DISCUSSION THREAD

**Discussion by gjoue** [ANTsX/ANTs#1282] [Q&A]
Hi,

I need to coregister high-resolution T2w-TSE slabs with whole-brain T1w-MPRAGEs so I can run ASHS, an automatic segmentation routine. For ASHS to work properly, I seem to need to retain the original dimensions/resolution of the images. Is there a way to do this now in ANTS? A search through the forums seems to tell me that this is not possible? 

This is how I am calling ANTS, where I am moving TSE slabs to their respective MPRAGE -- coregistration looks great, but the modified image resolution of the coregistered TSE adversely affects ASHS segmentations.

```
antsRegistration -d 3 \
                  -v \
                  -r [ $FIXED , $MOVING, 1] \
                  -o [ $OUTbase , ${OUTbase}.nii.gz ] \
                  -w [0.005,0.995] \ 
                  -u 0 \ 
                  -t Rigid[0.5] \
                  -m MI[ $FIXED, $MOVING, 1, 32, Regular, 0.2] \
                  -c [5000x1000x500, 1e-08, 10] \
                  -f 6x4x1 \
                  -s 3x2x0 \
		  -t Affine[0.1] \
                  -m MI[ $FIXED, $MOVING, 1, 32, Regular, 0.8] \
                  -c [1000x50x250x100, 1e-08, 10] \
                  -f 8x4x2x1 \
                  -s 4x2x1x0
```

Thanks for any help or suggestions,
Gina

**Comment by ntustison** [comment]
This sounds like a question for the ASHS developers (who we know well and are very familiar with ANTs).  Did you ask them?

**Reply by gjoue** [reply]
Posting a message at the mo' -- but irrespective of ASHS, is there a way to run antsRegistration so that the only the header of the registered image is modified but the image is not resampled?

**Reply by ntustison** [reply]
No.

**Comment by gdevenyi** [comment]
Such resampling is available in minc-land as `mincresample --tfm_input_sampling` and should only be used for rigid transformations (and can only be used for rigid/affine).
